base_layer/core/src/base_node/state_machine_service/states/events_and_states.rs

```diff
@@ -216,7 +216,7 @@ pub struct StatusInfo {
 impl StatusInfo {
     pub fn new() -> Self {
         Self {
-            bootstrapped: false,
+            bootstrapped: true,
```

Comment:
This doesn't make sense. The whole point of Bootstrapped is to indicate that the node has not synced even once since startup. That is why when you first start the node it is in the StartUp state and Bootstrapped = false. It should only flip to True once it has synced at least once.

Reply:
This is the code that should flip it to true inside the Listening State:
```Rust
if sync_mode.is_lagging() {
                        return StateEvent::FallenBehind(sync_mode);
                    } else {
                        if !shared.bootstrapped_sync && sync_mode == SyncStatus::UpToDate {
                            shared.bootstrapped_sync = true;
                            debug!(
                                target: LOG_TARGET,
                                "Initial sync achieved, bootstrap done: {}", sync_mode
                            );
                        }
                        self.is_synced = true;
                        shared.set_state_info(StateInfo::Listening(ListeningInfo::new(true)));
                    }
```

Reply:
I Think this might have slipped through. I have been changing values left and right to get it to work correctly